src/components/SearchBar.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import "./SearchBar.css";
+
+class SearchBar extends React.Component {
+  state = { searchTerm: "New York" };
+
+  onInputChange = event => {
+    this.setState({ searchTerm: event.target.value });
+  };
+
+  onFormSubmit = event => {
+    event.preventDefault();
+
+    this.props.onCitySubmit(this.state.searchTerm);
+  };
+
+  render() {
+    return (
+      <div className="search-bar">
+        <div className="ui centered row">
+          <form onSubmit={this.onFormSubmit} className="ui form">
+            <div className="ui fluid action input">
+              <input
+                type="text"
+                placeholder="City Search"
+                value={this.state.searchTerm}
+                onChange={this.onInputChange}
+              />
+              <div className="ui button">Search</div>
```

Comment:
[question] Does this button have any effect on the form? Currently, I am only able to submit using enter rather than that in addition to using this button. How might that be fixed?

Reply:
Ah whoops, that was just an oversight on my part. I had set up that button as a placeholder in the UI but forgot to wire up the functionality at the end. It just needs to pass in `this.props.onCitySubmit` as the handler to an onClick in that div.